capstone/src/main/webapp/style.css

```diff
@@ -0,0 +1,82 @@
+
+* {
+  box-sizing: border-box;
+}
+
+.column {
+    float: left;
+    padding: 10px;
```

Comment:
instead of using float, I think we should consider using flexbox for the entire thing... will save us a lot of time and make things a lot more powerful.... a great tutorial/explanation is here https://css-tricks.com/snippets/css/a-guide-to-flexbox/